lt lang

gymka opened the pull request.

Noitidart commented:
Thank you very much @gymka for this PR! Ideally I prefer if all my translators can make a PR from the work they did on babelzilla. This way the contribution shows up in your personal profile on github.

Why did you close this? Is it ok to merge it?

gymka commented:
too much files, i added and working files:) now added .gitignore file. i work on .po files, then convert them back to mozila format. i make new clean commit

Noitidart commented:
Oh I see, ok thank you!
